<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ATMECSINDT-052" ignored="0" name="Test" tests="8" failures="1" timestamp="21 May 2019 08:08:01 GMT" time="0.441" errors="0">
  <testcase name="testFinalBilling" time="0.01" classname="com.atmecs.bookmyshow.AppTest"/>
  <testcase name="testPaymentbMode" time="0.359" classname="com.atmecs.bookmyshow.AppTest"/>
  <testcase name="testRevenue" time="0.004" classname="com.atmecs.bookmyshow.AppTest"/>
  <testcase name="testSeatAvailability" time="0.003" classname="com.atmecs.bookmyshow.AppTest"/>
  <testcase name="testSeatSelection" time="0.001" classname="com.atmecs.bookmyshow.AppTest"/>
  <testcase name="testSeatStatus" time="0.001" classname="com.atmecs.bookmyshow.AppTest"/>
  <testcase name="testSeatVerification" time="0.002" classname="com.atmecs.bookmyshow.AppTest"/>
  <testcase name="testuserLoginSignUpChoice" time="0.012" classname="com.atmecs.bookmyshow.AppTest">
    <failure type="java.lang.AssertionError" message="expected [0] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [1]
at com.atmecs.bookmyshow.AppTest.testuserLoginSignUpChoice(AppTest.java:74)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testuserLoginSignUpChoice -->
</testsuite> <!-- Test -->
